README.md

```diff
@@ -1287,12 +1357,13 @@ A diagram with all the terms is likely worth it.
 🄳 Sometimes, to help you lower your tax burden, the company makes it possible to **early
 exercise (or forward exercise)** stock options.
 This means you exercise them even before they vest, which means you become a stockholder
-right away. This will have [tax implications](#83b-elections).
+earlier than you would otherwise.
+This will have [tax implications](#83b-elections).
 
-🔸 However, the company has the right to repurchase the *unvested* shares, at a lower
-price than what you paid, or the fair market value of the shares, if you quit working for
-the company. The company will typically repurchase the unvested shares should you leave
-the company before the stock you’ve purchased vests.
+🔸 However, the company has the right to repurchase the *unvested* shares, at the lower of
+the price or the fair market value of the shares, if you quit working for the company.
```

Comment:
"at the lower of the price"... Is that correct phrasing in finance? It does not have clear meaning to me. Why not "at a lower price than what you paid"?

Reply:
Yes, that's correct. It's a bit confusing but it means "the lower of X and Y" where X = "the price" and Y = "the fair market value of the shares".

The rewrite you suggest would mean something else, i.e. that they are paying some unspecified lower amount.

I'll change it to be clearer but still correct:

"... at the price paid or the fair market value of the shares (whichever is lower), if..."

Reply:
👍